integration/airflow/openlineage/airflow/listener.py

```diff
@@ -87,11 +88,12 @@ def on_task_instance_running(previous_state, task_instance: "TaskInstance", sess
     dag = task_instance.task.dag
 
     def on_running():
-        task_instance.render_templates()
-        task = task_instance.task
+        new_task_instance = copy.deepcopy(task_instance)
```

Comment:
if it is a deepcopy, we can create a variable name with `deepcopy` or `copy` suffix.